test/.eslintrc.json

```diff
@@ -1,4 +1,6 @@
 {
+	"extends": "../src/.eslintrc.json",
+	"root": true,
```

Comment:
We'd have to duplicate the globals in every `src`-related `.eslintrc.json`. What's the drawback of what I proposed?